torch/csrc/cuda/nccl.h

```diff
@@ -44,8 +44,9 @@ enum class ncclResult {
   InternalError = 3,
   InvalidArgument = 4,
   InvalidUsage = 5,
-  NumResults = 6,
```

Comment:
Is it possible to append RemoteError=8 without changing the previous values?

Reply:
[Here](https://github.com/NVIDIA/nccl/blob/master/src/nccl.h.in#L37-L45) are the official values.
While remapping (6<->8) can be an option, it can be confusing and may cause issues down the road.
Is there any issue with updating the previous values?

Reply:
OK the definition was copied from NCCL.  Is it better to drop our version and use the official definition?

Reply:
Yes, the nccl's official (doc)[https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/api/types.html#ncclresult-t] has the same values. My opinion is to use the official values.

Reply:
Try to define an alias enum by using?

Reply:
Do you suggest removing NumResults? Since the actual NNCL code still has the definition, it may be good to keep it just in case.

If this is not your intention, can you please provide the details so that I can act? Your comment is unclear.